Scope.Installer/Installer.cs

```diff
@@ -32,6 +40,36 @@ public static async Task Main(string[] args)
             try
             {
                 var download = await Download(DownloadUrl);
+                var archive = new ZipArchive(download);
+                if (archive.Entries.All(x => !x.FullName.StartsWith("ScopeStuff")))
```

Comment:
We gotta rely on the hash or something like that

Reply:
Yeah I know, but we should already think about production rather than dev branch

Reply:
We could bundle the scope files seperate from external libraries (bepinex, modded gameassembly, etc) and get the scope files from our domain, and the external ones from their respective releases